### DISCUSSION TITLE
付款一个多月了都无法授权使用专业版，赶紧解决一下

### DISCUSSION THREAD

**Discussion by bailang571** [getlantern/lantern-forum-cn#869] [General]
账号：bailang571@sina.com
很多年的老用户了，两个设备一起变成免费版，一个多月了都没能正常使用
![e100eaf07bd559f2ce04d2f989bd899](https://github.com/user-attachments/assets/bdf9424e-1723-4e26-9760-34826333d11f)

**Comment by MaxMKMax** [comment]
@bailang571 好的. 我帮您提交问题上去. 谢谢您!
